src/sentry/sentry_metrics/consumers/indexer/batch.py

```diff
@@ -123,7 +123,11 @@ def _extract_messages(self) -> None:
                 "sentry-metrics.indexer.disabled-namespaces"
             ):
                 self.skipped_offsets.add(partition_offset)
-                metrics.incr("process_messages.namespace_disabled", tags={"namespace": namespace})
+                metrics.incr(
+                    "process_messages.namespace_disabled",
+                    sample_rate=0.001,
```

Comment:
You do not want to add a sample rate. 
This means that you would be unlikely to log this metric on low volume. 

Keeping a local counter and periodically flush it would be more accurate.